test/Common.FunctionalTests/ConfigurationChangeTests.cs

```diff
@@ -54,5 +55,24 @@ public async Task ConfigurationChangeForcesChildProcessRestart(AncmVersion versi
             // Verify that worker process gets restarted with new process id
             await deploymentResult.HttpClient.RetryRequestAsync("/ProcessId", async r => await r.Content.ReadAsStringAsync() != processBefore);
         }
+
+        [ConditionalFact]
+        public async Task OutOfProcessToInProcessHostingModelSwitchWorks()
```

Comment:
What about inprocess to out of process and expect a new process to start.

Reply:
It won't start our testsites do not support switching hosting mode.